From d95700604fbc6fd7534303796d005e6efbd1d591 Mon Sep 17 00:00:00 2001
From: YiPeng Chai <YiPeng.Chai@amd.com>
Date: Thu, 30 Jul 2026 14:12:20 +0800
Subject: [PATCH 11301/15333] drm/amd/ras: rename core ras_* files and update
 include guards

Rename selected RAS core source and header filenames to drop
redundant ras_ prefixes in file names only.

Update related build object names, header include references, and
include guard macros in renamed headers.

Function/type prefixes and runtime behavior are unchanged.

No functional change intended.

Signed-off-by: YiPeng Chai <YiPeng.Chai@amd.com>
Reviewed-by: Hawking Zhang <Hawking.Zhang@amd.com>
Signed-off-by: Alex Deucher <alexander.deucher@amd.com>
---
 drivers/gpu/drm/amd/ras/core/Makefile              | 14 +++++++-------
 drivers/gpu/drm/amd/ras/core/{ras_aca.c => aca.c}  |  4 ++--
 drivers/gpu/drm/amd/ras/core/{ras_aca.h => aca.h}  |  4 ++--
 .../amd/ras/core/{ras_aca_v1_0.c => aca_v1_0.c}    |  6 +++---
 .../amd/ras/core/{ras_aca_v1_0.h => aca_v1_0.h}    |  4 ++--
 drivers/gpu/drm/amd/ras/core/{ras_cmd.c => cmd.c}  |  2 +-
 drivers/gpu/drm/amd/ras/core/{ras_cmd.h => cmd.h}  |  8 ++++----
 .../gpu/drm/amd/ras/core/{ras_core.c => core.c}    |  2 +-
 .../ras/core/{ras_core_status.h => core_status.h}  |  4 ++--
 .../drm/amd/ras/core/{ras_eeprom.c => eeprom.c}    |  2 +-
 .../drm/amd/ras/core/{ras_eeprom.h => eeprom.h}    |  4 ++--
 .../amd/ras/core/{ras_eeprom_fw.c => eeprom_fw.c}  |  0
 .../amd/ras/core/{ras_eeprom_fw.h => eeprom_fw.h}  |  4 ++--
 .../amd/ras/core/{ras_log_ring.c => log_ring.c}    |  4 ++--
 .../amd/ras/core/{ras_log_ring.h => log_ring.h}    |  6 +++---
 drivers/gpu/drm/amd/ras/core/ras.h                 | 12 ++++++------
 drivers/gpu/drm/amd/ras/core/ras_cper.c            |  4 ++--
 drivers/gpu/drm/amd/ras/core/ras_gfx.c             |  2 +-
 drivers/gpu/drm/amd/ras/core/ras_gfx_v9_0.c        |  2 +-
 drivers/gpu/drm/amd/ras/core/ras_mp1_v13_0.c       |  2 +-
 drivers/gpu/drm/amd/ras/core/ras_psp.c             |  2 +-
 drivers/gpu/drm/amd/ras/core/ras_psp.h             |  2 +-
 drivers/gpu/drm/amd/ras/core/ras_umc.h             |  4 ++--
 drivers/gpu/drm/amd/ras/core/ras_umc_v12_0.c       |  2 +-
 .../gpu/drm/amd/ras/core/{ras_ta_if.h => ta_if.h}  |  4 ++--
 .../drm/amd/ras/ras_mgr/amdgpu_ras_eeprom_i2c.c    |  2 +-
 26 files changed, 53 insertions(+), 53 deletions(-)
 rename drivers/gpu/drm/amd/ras/core/{ras_aca.c => aca.c} (99%)
 rename drivers/gpu/drm/amd/ras/core/{ras_aca.h => aca.h} (98%)
 rename drivers/gpu/drm/amd/ras/core/{ras_aca_v1_0.c => aca_v1_0.c} (99%)
 rename drivers/gpu/drm/amd/ras/core/{ras_aca_v1_0.h => aca_v1_0.h} (98%)
 rename drivers/gpu/drm/amd/ras/core/{ras_cmd.c => cmd.c} (99%)
 rename drivers/gpu/drm/amd/ras/core/{ras_cmd.h => cmd.h} (99%)
 rename drivers/gpu/drm/amd/ras/core/{ras_core.c => core.c} (99%)
 rename drivers/gpu/drm/amd/ras/core/{ras_core_status.h => core_status.h} (96%)
 rename drivers/gpu/drm/amd/ras/core/{ras_eeprom.c => eeprom.c} (99%)
 rename drivers/gpu/drm/amd/ras/core/{ras_eeprom.h => eeprom.h} (99%)
 rename drivers/gpu/drm/amd/ras/core/{ras_eeprom_fw.c => eeprom_fw.c} (100%)
 rename drivers/gpu/drm/amd/ras/core/{ras_eeprom_fw.h => eeprom_fw.h} (98%)
 rename drivers/gpu/drm/amd/ras/core/{ras_log_ring.c => log_ring.c} (99%)
 rename drivers/gpu/drm/amd/ras/core/{ras_log_ring.h => log_ring.h} (97%)
 rename drivers/gpu/drm/amd/ras/core/{ras_ta_if.h => ta_if.h} (99%)

diff --git a/drivers/gpu/drm/amd/ras/core/Makefile b/drivers/gpu/drm/amd/ras/core/Makefile
index 454f83f62930..5b4cd6a3d358 100644
--- a/drivers/gpu/drm/amd/ras/core/Makefile
+++ b/drivers/gpu/drm/amd/ras/core/Makefile
@@ -19,25 +19,25 @@
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
 #
-RAS_CORE_FILES = ras_core.o \
+RAS_CORE_FILES = core.o \
 			ras_mp1.o \
 			ras_mp1_v13_0.o \
-			ras_aca.o  \
-			ras_aca_v1_0.o \
-			ras_eeprom.o \
+			aca.o  \
+			aca_v1_0.o \
+			eeprom.o \
 			ras_umc.o \
 			ras_umc_v12_0.o \
-			ras_cmd.o \
+			cmd.o \
 			ras_gfx.o \
 			ras_gfx_v9_0.o \
 			ras_process.o \
 			ras_nbio.o \
 			ras_nbio_v7_9.o \
-			ras_log_ring.o \
+			log_ring.o \
 			ras_cper.o \
 			ras_psp.o \
 			ras_psp_v13_0.o \
-			ras_eeprom_fw.o
+			eeprom_fw.o
 
 
 RAS_CORE = $(addprefix $(AMD_GPU_RAS_PATH)/core/,$(RAS_CORE_FILES))
diff --git a/drivers/gpu/drm/amd/ras/core/ras_aca.c b/drivers/gpu/drm/amd/ras/core/aca.c
similarity index 99%
rename from drivers/gpu/drm/amd/ras/core/ras_aca.c
rename to drivers/gpu/drm/amd/ras/core/aca.c
index 6f57b5d09cd3..fa7cb3277578 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_aca.c
+++ b/drivers/gpu/drm/amd/ras/core/aca.c
@@ -22,8 +22,8 @@
  *
  */
 #include "ras.h"
-#include "ras_aca.h"
-#include "ras_aca_v1_0.h"
+#include "aca.h"
+#include "aca_v1_0.h"
 #include "ras_mp1_v13_0.h"
 
 #define ACA_MARK_FATAL_FLAG    0x100
diff --git a/drivers/gpu/drm/amd/ras/core/ras_aca.h b/drivers/gpu/drm/amd/ras/core/aca.h
similarity index 98%
rename from drivers/gpu/drm/amd/ras/core/ras_aca.h
rename to drivers/gpu/drm/amd/ras/core/aca.h
index f61b02a5f0fc..3c1e691cd2ee 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_aca.h
+++ b/drivers/gpu/drm/amd/ras/core/aca.h
@@ -22,8 +22,8 @@
  *
  */
 
-#ifndef __RAS_ACA_H__
-#define __RAS_ACA_H__
+#ifndef __ACA_H__
+#define __ACA_H__
 #include "ras.h"
 
 #define MAX_SOCKET_NUM_PER_HIVE 8
diff --git a/drivers/gpu/drm/amd/ras/core/ras_aca_v1_0.c b/drivers/gpu/drm/amd/ras/core/aca_v1_0.c
similarity index 99%
rename from drivers/gpu/drm/amd/ras/core/ras_aca_v1_0.c
rename to drivers/gpu/drm/amd/ras/core/aca_v1_0.c
index 288fad0862a2..a8b413270de3 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_aca_v1_0.c
+++ b/drivers/gpu/drm/amd/ras/core/aca_v1_0.c
@@ -22,9 +22,9 @@
  *
  */
 #include "ras.h"
-#include "ras_aca.h"
-#include "ras_core_status.h"
-#include "ras_aca_v1_0.h"
+#include "aca.h"
+#include "core_status.h"
+#include "aca_v1_0.h"
 
 struct ras_aca_hwip {
 	int hwid;
diff --git a/drivers/gpu/drm/amd/ras/core/ras_aca_v1_0.h b/drivers/gpu/drm/amd/ras/core/aca_v1_0.h
similarity index 98%
rename from drivers/gpu/drm/amd/ras/core/ras_aca_v1_0.h
rename to drivers/gpu/drm/amd/ras/core/aca_v1_0.h
index 40e5d94b037f..f18bdf043b62 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_aca_v1_0.h
+++ b/drivers/gpu/drm/amd/ras/core/aca_v1_0.h
@@ -22,8 +22,8 @@
  *
  */
 
-#ifndef __RAS_ACA_V1_0_H__
-#define __RAS_ACA_V1_0_H__
+#ifndef __ACA_V1_0_H__
+#define __ACA_V1_0_H__
 #include "ras.h"
 
 #define ACA__REG__FIELD(x, h, l)		  (((x) & GENMASK_ULL(h, l)) >> l)
diff --git a/drivers/gpu/drm/amd/ras/core/ras_cmd.c b/drivers/gpu/drm/amd/ras/core/cmd.c
similarity index 99%
rename from drivers/gpu/drm/amd/ras/core/ras_cmd.c
rename to drivers/gpu/drm/amd/ras/core/cmd.c
index abd004eee95e..6c37dc519eed 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_cmd.c
+++ b/drivers/gpu/drm/amd/ras/core/cmd.c
@@ -22,7 +22,7 @@
  *
  */
 #include "ras.h"
-#include "ras_cmd.h"
+#include "cmd.h"
 
 #define RAS_CMD_MAJOR_VERSION 6
 #define RAS_CMD_MINOR_VERSION 0
diff --git a/drivers/gpu/drm/amd/ras/core/ras_cmd.h b/drivers/gpu/drm/amd/ras/core/cmd.h
similarity index 99%
rename from drivers/gpu/drm/amd/ras/core/ras_cmd.h
rename to drivers/gpu/drm/amd/ras/core/cmd.h
index c376e074e15d..e3b39866455d 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_cmd.h
+++ b/drivers/gpu/drm/amd/ras/core/cmd.h
@@ -22,11 +22,11 @@
  *
  */
 
-#ifndef __RAS_CMD_H__
-#define __RAS_CMD_H__
+#ifndef __CMD_H__
+#define __CMD_H__
 #include "ras.h"
-#include "ras_eeprom.h"
-#include "ras_log_ring.h"
+#include "eeprom.h"
+#include "log_ring.h"
 #include "ras_cper.h"
 
 #define RAS_CMD_MAX_IN_SIZE 256
diff --git a/drivers/gpu/drm/amd/ras/core/ras_core.c b/drivers/gpu/drm/amd/ras/core/core.c
similarity index 99%
rename from drivers/gpu/drm/amd/ras/core/ras_core.c
rename to drivers/gpu/drm/amd/ras/core/core.c
index 75e8216fae8a..0e19cf41bc26 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_core.c
+++ b/drivers/gpu/drm/amd/ras/core/core.c
@@ -22,7 +22,7 @@
  *
  */
 #include "ras.h"
-#include "ras_core_status.h"
+#include "core_status.h"
 
 #define RAS_SEQNO_FIFO_SIZE (128 * sizeof(uint64_t))
 
diff --git a/drivers/gpu/drm/amd/ras/core/ras_core_status.h b/drivers/gpu/drm/amd/ras/core/core_status.h
similarity index 96%
rename from drivers/gpu/drm/amd/ras/core/ras_core_status.h
rename to drivers/gpu/drm/amd/ras/core/core_status.h
index 144fbe4ceb9a..f91e185fad5c 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_core_status.h
+++ b/drivers/gpu/drm/amd/ras/core/core_status.h
@@ -22,8 +22,8 @@
  *
  */
 
-#ifndef __RAS_CORE_STATUS_H__
-#define __RAS_CORE_STATUS_H__
+#ifndef __CORE_STATUS_H__
+#define __CORE_STATUS_H__
 
 #define RAS_CORE_OK                       0
 #define RAS_CORE_NOT_SUPPORTED            248
diff --git a/drivers/gpu/drm/amd/ras/core/ras_eeprom.c b/drivers/gpu/drm/amd/ras/core/eeprom.c
similarity index 99%
rename from drivers/gpu/drm/amd/ras/core/ras_eeprom.c
rename to drivers/gpu/drm/amd/ras/core/eeprom.c
index 3b1883a04179..7afaac81f37a 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_eeprom.c
+++ b/drivers/gpu/drm/amd/ras/core/eeprom.c
@@ -22,7 +22,7 @@
  *
  */
 
-#include "ras_eeprom.h"
+#include "eeprom.h"
 #include "ras.h"
 
 /* These are memory addresses as would be seen by one or more EEPROM
diff --git a/drivers/gpu/drm/amd/ras/core/ras_eeprom.h b/drivers/gpu/drm/amd/ras/core/eeprom.h
similarity index 99%
rename from drivers/gpu/drm/amd/ras/core/ras_eeprom.h
rename to drivers/gpu/drm/amd/ras/core/eeprom.h
index 5c3e92f710b4..32bd1526bc55 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_eeprom.h
+++ b/drivers/gpu/drm/amd/ras/core/eeprom.h
@@ -22,8 +22,8 @@
  *
  */
 
-#ifndef __RAS_EEPROM_H__
-#define __RAS_EEPROM_H__
+#ifndef __EEPROM_H__
+#define __EEPROM_H__
 #include "ras_sys.h"
 
 #define RAS_TABLE_VER_V1           0x00010000
diff --git a/drivers/gpu/drm/amd/ras/core/ras_eeprom_fw.c b/drivers/gpu/drm/amd/ras/core/eeprom_fw.c
similarity index 100%
rename from drivers/gpu/drm/amd/ras/core/ras_eeprom_fw.c
rename to drivers/gpu/drm/amd/ras/core/eeprom_fw.c
diff --git a/drivers/gpu/drm/amd/ras/core/ras_eeprom_fw.h b/drivers/gpu/drm/amd/ras/core/eeprom_fw.h
similarity index 98%
rename from drivers/gpu/drm/amd/ras/core/ras_eeprom_fw.h
rename to drivers/gpu/drm/amd/ras/core/eeprom_fw.h
index 762345be075c..803d3ccdaddb 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_eeprom_fw.h
+++ b/drivers/gpu/drm/amd/ras/core/eeprom_fw.h
@@ -21,8 +21,8 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  *
  */
-#ifndef __RAS_EEPROM_FW_H__
-#define __RAS_EEPROM_FW_H__
+#ifndef __EEPROM_FW_H__
+#define __EEPROM_FW_H__
 
 struct ras_fw_eeprom_control {
 	uint32_t version;
diff --git a/drivers/gpu/drm/amd/ras/core/ras_log_ring.c b/drivers/gpu/drm/amd/ras/core/log_ring.c
similarity index 99%
rename from drivers/gpu/drm/amd/ras/core/ras_log_ring.c
rename to drivers/gpu/drm/amd/ras/core/log_ring.c
index c37343db39df..175c8b357144 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_log_ring.c
+++ b/drivers/gpu/drm/amd/ras/core/log_ring.c
@@ -22,8 +22,8 @@
  *
  */
 #include "ras.h"
-#include "ras_core_status.h"
-#include "ras_log_ring.h"
+#include "core_status.h"
+#include "log_ring.h"
 
 #define RAS_LOG_MAX_QUERY_SIZE   0xC000
 #define RAS_LOG_MEM_TEMP_SIZE    0x200
diff --git a/drivers/gpu/drm/amd/ras/core/ras_log_ring.h b/drivers/gpu/drm/amd/ras/core/log_ring.h
similarity index 97%
rename from drivers/gpu/drm/amd/ras/core/ras_log_ring.h
rename to drivers/gpu/drm/amd/ras/core/log_ring.h
index cb66beaa9f43..a1ef2f24130c 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_log_ring.h
+++ b/drivers/gpu/drm/amd/ras/core/log_ring.h
@@ -21,9 +21,9 @@
  * OTHER DEALINGS IN THE SOFTWARE.
  *
  */
-#ifndef __RAS_LOG_RING_H__
-#define __RAS_LOG_RING_H__
-#include "ras_aca.h"
+#ifndef __LOG_RING_H__
+#define __LOG_RING_H__
+#include "aca.h"
 
 #define MAX_RECORD_PER_BATCH 32
 
diff --git a/drivers/gpu/drm/amd/ras/core/ras.h b/drivers/gpu/drm/amd/ras/core/ras.h
index 8c3d603ba801..b4e40d7cc0e0 100644
--- a/drivers/gpu/drm/amd/ras/core/ras.h
+++ b/drivers/gpu/drm/amd/ras/core/ras.h
@@ -26,17 +26,17 @@
 #define __RAS_H__
 #include "ras_sys.h"
 #include "ras_umc.h"
-#include "ras_aca.h"
-#include "ras_eeprom.h"
-#include "ras_core_status.h"
+#include "aca.h"
+#include "eeprom.h"
+#include "core_status.h"
 #include "ras_process.h"
 #include "ras_gfx.h"
-#include "ras_cmd.h"
+#include "cmd.h"
 #include "ras_nbio.h"
 #include "ras_mp1.h"
 #include "ras_psp.h"
-#include "ras_log_ring.h"
-#include "ras_eeprom_fw.h"
+#include "log_ring.h"
+#include "eeprom_fw.h"
 
 #define RAS_HW_ERR		"[Hardware Error]: "
 
diff --git a/drivers/gpu/drm/amd/ras/core/ras_cper.c b/drivers/gpu/drm/amd/ras/core/ras_cper.c
index 6e93a13bbc4c..4ffac4289dbd 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_cper.c
+++ b/drivers/gpu/drm/amd/ras/core/ras_cper.c
@@ -22,8 +22,8 @@
  *
  */
 #include "ras.h"
-#include "ras_core_status.h"
-#include "ras_log_ring.h"
+#include "core_status.h"
+#include "log_ring.h"
 #include "ras_cper.h"
 
 static const struct ras_cper_guid MCE	= CPER_NOTIFY__MCE;
diff --git a/drivers/gpu/drm/amd/ras/core/ras_gfx.c b/drivers/gpu/drm/amd/ras/core/ras_gfx.c
index f5ce28777705..2efe18cbbe5b 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_gfx.c
+++ b/drivers/gpu/drm/amd/ras/core/ras_gfx.c
@@ -25,7 +25,7 @@
 #include "ras.h"
 #include "ras_gfx_v9_0.h"
 #include "ras_gfx.h"
-#include "ras_core_status.h"
+#include "core_status.h"
 
 static const struct ras_gfx_ip_func *ras_gfx_get_ip_funcs(
 				struct ras_core_context *ras_core, uint32_t ip_version)
diff --git a/drivers/gpu/drm/amd/ras/core/ras_gfx_v9_0.c b/drivers/gpu/drm/amd/ras/core/ras_gfx_v9_0.c
index 6213d3f125be..6fb3ec1d102d 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_gfx_v9_0.c
+++ b/drivers/gpu/drm/amd/ras/core/ras_gfx_v9_0.c
@@ -23,7 +23,7 @@
  */
 #include "ras.h"
 #include "ras_gfx_v9_0.h"
-#include "ras_core_status.h"
+#include "core_status.h"
 
 enum ta_gfx_v9_subblock {
 	/*CPC*/
diff --git a/drivers/gpu/drm/amd/ras/core/ras_mp1_v13_0.c b/drivers/gpu/drm/amd/ras/core/ras_mp1_v13_0.c
index 1fcfc1995ad3..b9db91859e5d 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_mp1_v13_0.c
+++ b/drivers/gpu/drm/amd/ras/core/ras_mp1_v13_0.c
@@ -23,7 +23,7 @@
  */
 #include "ras.h"
 #include "ras_mp1.h"
-#include "ras_core_status.h"
+#include "core_status.h"
 #include "ras_mp1_v13_0.h"
 
 #define RAS_MP1_MSG_QueryValidMcaCount                0x36
diff --git a/drivers/gpu/drm/amd/ras/core/ras_psp.c b/drivers/gpu/drm/amd/ras/core/ras_psp.c
index 35264c119bbb..9bf20201649f 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_psp.c
+++ b/drivers/gpu/drm/amd/ras/core/ras_psp.c
@@ -22,7 +22,7 @@
  *
  */
 #include "ras.h"
-#include "ras_ta_if.h"
+#include "ta_if.h"
 #include "ras_psp.h"
 #include "ras_psp_v13_0.h"
 
diff --git a/drivers/gpu/drm/amd/ras/core/ras_psp.h b/drivers/gpu/drm/amd/ras/core/ras_psp.h
index 347f5334c3f3..c899d0ac74cc 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_psp.h
+++ b/drivers/gpu/drm/amd/ras/core/ras_psp.h
@@ -24,7 +24,7 @@
 #ifndef __RAS_PSP_H__
 #define __RAS_PSP_H__
 #include "ras.h"
-#include "ras_ta_if.h"
+#include "ta_if.h"
 
 struct ras_core_context;
 struct ras_ta_trigger_error_input;
diff --git a/drivers/gpu/drm/amd/ras/core/ras_umc.h b/drivers/gpu/drm/amd/ras/core/ras_umc.h
index 3d902b0d663a..bf14036a4aaa 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_umc.h
+++ b/drivers/gpu/drm/amd/ras/core/ras_umc.h
@@ -25,8 +25,8 @@
 #ifndef __RAS_UMC_H__
 #define __RAS_UMC_H__
 #include "ras.h"
-#include "ras_eeprom.h"
-#include "ras_cmd.h"
+#include "eeprom.h"
+#include "cmd.h"
 
 #define UMC_VRAM_TYPE_UNKNOWN 0
 #define UMC_VRAM_TYPE_GDDR1   1
diff --git a/drivers/gpu/drm/amd/ras/core/ras_umc_v12_0.c b/drivers/gpu/drm/amd/ras/core/ras_umc_v12_0.c
index 6bf30dee8d17..c7cf925a8345 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_umc_v12_0.c
+++ b/drivers/gpu/drm/amd/ras/core/ras_umc_v12_0.c
@@ -23,7 +23,7 @@
  */
 #include "ras.h"
 #include "ras_umc.h"
-#include "ras_core_status.h"
+#include "core_status.h"
 #include "ras_umc_v12_0.h"
 
 #define NumDieInterleaved 4
diff --git a/drivers/gpu/drm/amd/ras/core/ras_ta_if.h b/drivers/gpu/drm/amd/ras/core/ta_if.h
similarity index 99%
rename from drivers/gpu/drm/amd/ras/core/ras_ta_if.h
rename to drivers/gpu/drm/amd/ras/core/ta_if.h
index e910a75b3022..6b05c82679c0 100644
--- a/drivers/gpu/drm/amd/ras/core/ras_ta_if.h
+++ b/drivers/gpu/drm/amd/ras/core/ta_if.h
@@ -22,8 +22,8 @@
  *
  */
 
-#ifndef _RAS_TA_IF_H
-#define _RAS_TA_IF_H
+#ifndef _TA_IF_H
+#define _TA_IF_H
 #include "ras.h"
 
 #define RAS_TA_HOST_IF_VER	0
diff --git a/drivers/gpu/drm/amd/ras/ras_mgr/amdgpu_ras_eeprom_i2c.c b/drivers/gpu/drm/amd/ras/ras_mgr/amdgpu_ras_eeprom_i2c.c
index 9c6d0024210d..aa97b58c9422 100644
--- a/drivers/gpu/drm/amd/ras/ras_mgr/amdgpu_ras_eeprom_i2c.c
+++ b/drivers/gpu/drm/amd/ras/ras_mgr/amdgpu_ras_eeprom_i2c.c
@@ -26,7 +26,7 @@
 #include "amdgpu_ras_eeprom.h"
 #include "amdgpu_ras_mgr.h"
 #include "amdgpu_ras_eeprom_i2c.h"
-#include "ras_eeprom.h"
+#include "eeprom.h"
 
 /* These are memory addresses as would be seen by one or more EEPROM
  * chips strung on the I2C bus, usually by manipulating pins 1-3 of a
-- 
2.55.0

